hivemq4/k8s-image/Dockerfile

```diff
@@ -1,46 +1,39 @@
 ARG BASEIMAGE=hivemq/hivemq4:dns-latest
-ARG PROMETHEUS_EXTENSION_VERSION=4.0.8
+
 
 FROM alpine:3.19@sha256:c5b1261d6d3e43071626931fc004f70149baeba2c8ec672bd4f27761f8e1ad6b as BUILD
-ARG PROMETHEUS_EXTENSION_VERSION
 
-WORKDIR /opt/hivemq/extensions
+ARG PROMETHEUS_EXTENSION_VERSION=4.0.8
+
+RUN mkdir -p /opt/hivemq/extensions
```

Comment:
I think we can avoid adding this layer since the `unzip` below creates it.

Reply:
This is not the final image, it is only an intermediate build stage.

Reply:
Ofc sorry for the noise.